.env.neo4j.example

```diff
@@ -1,11 +1,11 @@
-NEO4J_dbms_memory_heap_initial__size=512M
-NEO4J_dbms_memory_heap_max__size=512M
-NEO4J_dbms_memory_pagecache_size=512M
+NEO4J_server_memory_heap_initial__size=512M
+NEO4J_server_memory_heap_max__size=512M
+NEO4J_server_memory_pagecache_size=512M
+
```

Comment:
so actualy this will only work in v5

Reply:
oops, v4 was started but yep these variables were not taken in account, redoing a pr